contrib/lib/src/uuid.rs

```diff
@@ -64,6 +64,7 @@ type ParseError = <self::uuid_crate::Uuid as FromStr>::Err;
 ///     format!("User ID: {}", id)
 /// }
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize), serde(transparent))]
```

Comment:
Given that the `serde` feature is enabled if this code is being compiled, is this `cfg_attr` a no-op?

Reply:
We can not name it `uuid` as it would hit https://github.com/rust-lang/cargo/issues/1286#issue-57144439. What should we name it then? `uuid_integration` or `uuid_serde` or something else?

Reply:
You can indeed name it `uuid` as long as the crate is named something else. See, for example, how `rocket_http` does this for its `serde` feature:

https://github.com/SergioBenitez/Rocket/blob/3f56490663a395d348447918ff09f20ba9c83efc/core/http/Cargo.toml#L17-L21

https://github.com/SergioBenitez/Rocket/blob/3f56490663a395d348447918ff09f20ba9c83efc/core/http/Cargo.toml#L48-L53